### DISCUSSION TITLE
folder needs to exist for sqlitefile to be created

### DISCUSSION THREAD

**Discussion by AlphaMoonbaseBerlin** [payloadcms/payload#8771] [Feature Requests & Ideas]
### Environment Info

Windows,
NPM

### Describe the Bug

When running npx create-payload-app@beta and passing a path to the .db-file that contains a subfolder, the subfolder needs to exist for payload to create the file on init.

### Reproduction Steps

Run npx create-payload-app@beta
Select SQLite as the database.
Pass ```./db/dbfile.db``` as the path fo the file.
Opening localhost:3000/admin will crash as there is no connection to the db possible as the file does not exist.

**Comment by r1tsuu** [comment]
We're simply relying on how the underlying package for SQLite works and it doesn't create a folder automatically, therefore it's not a bug with Payload.

However, it may be a useful feature to have in the future, I'm converting this to a discussion.
